src/main/scala/org/clulab/habitus/printer/MentionInfo.scala

```diff
@@ -1,17 +1,24 @@
 package org.clulab.habitus.printer
 
+import com.typesafe.config.ConfigFactory
+import ai.lum.common.ConfigUtils._
 import org.clulab.habitus.utils.Pairable
 import org.clulab.odin.Mention
 
-case class MentionInfo(sentenceText: String, inputFilename: String, label: String) extends Pairable
+case class MentionInfo(contextWindow: String, sentenceText: String, inputFilename: String, label: String) extends Pairable
 
 object MentionInfo {
 
+  val config = ConfigFactory.load()
+  val windowSize: Int = config[Int]("VarDatesReader.contextWindowSize")
+
   def apply(mention: Mention, inputFilename: String): MentionInfo = {
     val document = mention.document
+    val sentInterval = (math.max(mention.sentence - windowSize, 0), math.min(mention.sentence + windowSize + 1, mention.document.sentences.length + 1))
```

Comment:
I *think* there might be something off on the right side of this interval.  Isn't it usually math.min(..., _.length)?  length is typically already one beyond the indexable values and adding another would be out of bounds.

Reply:
Ah right thanks!

Reply:
Around this line, `mention.document` could just be `document`, but see the next comment.  One could drill down directly to the sentences.